### ISSUE TITLE
multimode networks plugin

### ISSUE THREAD

**Issue by joancf** [gephi/gephi-plugins#130]
Some time ago I did some improvements to the multimode network plugins:

1.   Better progress bar, that was also progressing during matrix computation
2.  Possibility to filter out edges under a given threshold to aviod the number of edges  to be NxN
3. Different ways to compute the weight of the edge between nodes ( adding cosine distance ) 
I sent a pull request many time ago to the author of the plugin, but my code was not included so now I would like to incorporate it to the new plugin for 9.0 but ... I don't know how to find the code in order to do the merge.... 
Thanks

**Comment by mbastian** [comment]
Hi @joancf and sorry for the delay I'm working on cleaning things up over here. 

I'm piloting a slightly different process for those plugins that have changed hands and are maintained by multiple people. I've created a `multimode-plugin` branch on this repository based on the current code. I've also sent you an invitation to join a GitHub team. This team of Plugin Developers has permissions to push to these branches. 

So in summary, if you want to make changes you don't need to fork the repository anymore but rather:
- Clone this repo and checkout the `multimode-plugin` branch
- Make changes, commit and push (to this repo)
- Submit PR to merge into `master-forge`
